web/app/configs/capacities.json

```diff
@@ -239,6 +245,13 @@
             "wind": 2938
         }
     },
+    "RS": {
+        "capacity": {
+            "coal": 5644,
+            "hydro": 3006,
+            "unknown": 428
```

Comment:
"Unknown" is actually the "Panonske TE-TO power plant", which is a modern (2015) mixed Gas & Oil powered Combined Heat & Power plant, generating 424 MWe and 505 MWth in paralell. Most of its primary fuel is Gas. I suggest to consider it as Gas-electricity, with an IPCC median value for emission intensity of 490gCO2eq/kWhe. We will not account for the "free" heat but for the moment, heat is out of the scope of electricity map.